crates/project/src/lsp_store.rs

```diff
@@ -1553,10 +1559,20 @@ impl LocalLspStore {
                             cx,
                         )
                         .await
-                        .context("Failed to format ranges via language server")?
+                        {
+                            Ok(edits) => edits,
+                            Err(err) => {
+                                zlog::warn!(
+                                    logger =>
+                                    "Failed to format ranges via LSP: {}",
```

Comment:
```suggestion
                                    "Failed to format ranges via LSP: {e:#}",
```

Could be something like this, `:#` in `anyhow` library prints the entire chain of contexts lead to this failure, which helps with tracing the root cause.

We should also include the language server name and ID while we're at it.
Same below.

Reply:
> We should also include the language server name and ID while we're at it.
> Same below.